src/components/Search.tsx

```diff
@@ -109,22 +117,25 @@ const SearchContent = () => {
             'w-full appearance-none rounded shadow-cardLight border border-grayAlpha-200 p-2 text-sm outline-none hover:border-grayAlpha-300 focus:border-grayAlpha-400',
         }}
       />
-      {query && hits.hits.length > 0
-        ? hits.hits.map((hit, i) => (
-            <SearchResult
-              hit={hit as unknown as SearchHit}
-              active={i === currentIndex}
-              key={hit.objectID}
-            />
-          ))
-        : null}
-      {query && hits.hits.length === 0 ? (
-        <div className="flex items-center justify-center p-2 mt-2">
-          <span className="text-grayAlpha-400">
-            No results found for <strong>{query}</strong>
-          </span>
-        </div>
-      ) : null}
+      {query && hits?.hits.length > 0 ? <div className="h-4"></div> : null}
+      <div className="overflow-auto max-h-[calc(100vh-10rem)] flex flex-col gap-1">
+        {query && hits.hits.length > 0
+          ? hits.hits.map((hit, i) => (
+              <SearchResult
+                hit={hit as unknown as SearchHit}
```

Comment:
Type casting might lead to an error but if you're aware of it then it's fine

Reply:
Yeah - there are no official types for the BaseHit so we have to typecast it to at least have any typings in the `SearchResult` component to work with.